dev-libs/libopenrazer/libopenrazer-9999.ebuild

```
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit git-r3 meson
+
+DESCRIPTION="Qt wrapper around the D-Bus API from OpenRazer"
+HOMEPAGE="https://github.com/z3ntu/libopenrazer"
+EGIT_REPO_URI="https://github.com/z3ntu/libopenrazer.git"
+EGIT_CLONE_TYPE="shallow"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-linux"
+
+RDEPEND="
+	app-misc/razer-test
```

I've looked at `libopenrazer`, it seems to have a dependency on openrazer as well. Not sure if it's optional or required, either way, we should add a USE flag here as well.

---

Good catch!

---

Unlike razergenie, this does not seem to be a link-time dependency, however I think we should just add a hard dep until we fix razergenie as well.

```suggestion
	app-misc/openrazer[daemon]
	app-misc/razer-test
```